core.go

```diff
@@ -745,6 +745,36 @@ func ExtractChannel(src Mat, dst Mat, coi int) {
 	C.Mat_ExtractChannel(src.p, dst.p, C.int(coi))
 }
 
+// FindNonZero returns the list of locations of non-zero pixels
+//
+// For further details, please see:
+// https://docs.opencv.org/master/d2/de8/group__core__array.html#gaed7df59a3539b4cc0fe5c9c8d7586190
+//
+func FindNonZero(src Mat, idx Mat) {
+	C.Mat_FindNonZero(src.p, idx.p)
+}
+
+// Flip flips a 2D array around vertical, horizontal, or both axes.
+//
+// FlipCode	a flag to specify how to flip the array; 0 means flipping around the x-axis and positive value (for example, 1) means flipping around y-axis.
+// Negative value (for example, -1) means flipping around both axes.
+//
+// For further details, please see:
+// https://docs.opencv.org/master/d2/de8/group__core__array.html#gaca7be533e3dac7feb70fc60635adf441
+//
+func Flip(src Mat, dst Mat, flipCode int) {
```

Comment:
here too you can skip the first type since the two first arguments have the same type, Go will pick the type after the second argument.

Reply:
I wont repeat the comment but it applies to the whole file :)

Reply:
understood. my generator isn't yet up to it, and the rest of the codebase isn't too.

Reply:
oh that's smart, so you're using a generator to port all these functions?

Reply:
lol, i'm trying hard, but it only gets 80% right.

(at least, i can be sure it does not miss params)
(combination of copy/paste stuff done by a machine (they *are* better at it !) and human revision (comments, tests) for now.)